launchable/test_runners/cucumber.py

```diff
@@ -180,12 +222,19 @@ def parse_func(self, report_file: str) -> Generator[CaseEventType, None, None]:
             class_name = d.get("name", "")
             for element in d.get("elements", []):
                 test_case = element.get("name", "")
+                scenario_hook_information = _extract_test_case_info_from_hook(element)
```

Comment:
Question: How about using XML report file case? Is it already included??

Reply:
This is TEST-features-is_it_friday_yet.xml

```xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuite failures="1" errors="0" skipped="0" tests="6" time="0.526763" name="Is it Friday yet?">
<testcase classname="Is it Friday yet?" name="Today is or is not Friday (outline example : | Friday | TGIF |)" time="0.088964">
  <system-out>
    <![CDATA[output: 2023
output: January
]]>
  </system-out>
  <system-err>
    <![CDATA[]]>
  </system-err>
</testcase>
<testcase classname="Is it Friday yet?" name="Today is or is not Friday (outline example : | Sunday | Nope |)" time="0.088768">
  <system-out>
    <![CDATA[output: 2023
output: January
]]>
  </system-out>
  <system-err>
    <![CDATA[]]>
  </system-err>
</testcase>
<testcase classname="Is it Friday yet?" name="Today is or is not Friday (outline example : | anything else! | Nope |)" time="0.089323">
  <system-out>
    <![CDATA[output: 2023
output: January
]]>
  </system-out>
  <system-err>
    <![CDATA[]]>
  </system-err>
</testcase>
<testcase classname="Is it Friday yet?" name="Sunday isn't Friday" time="0.090021">
  <system-out>
    <![CDATA[output: 2023
output: January
]]>
  </system-out>
  <system-err>
    <![CDATA[]]>
  </system-err>
</testcase>
<testcase classname="Is it Friday yet?" name="Friday is Friday" time="0.090236">
  <system-out>
    <![CDATA[output: 2023
output: January
]]>
  </system-out>
  <system-err>
    <![CDATA[]]>
  </system-err>
</testcase>
<testcase classname="Is it Friday yet?" name="This is fail" time="0.079451">
  <failure message="failed This is fail" type="failed">
    <![CDATA[Scenario: This is fail

Then I should be told "Nope"

Message:
]]>
    <![CDATA[
expected: "Nope"
     got: "TGIF"

(compared using ==)
 (RSpec::Expectations::ExpectationNotMetError)
./features/step_definitions/stepdefs.rb:39:in `nil'
features/is_it_friday_yet.feature:32:in `I should be told "Nope"']]>
  </failure>
  <system-out>
    <![CDATA[output: 2023
output: January
]]>
  </system-out>
  <system-err>
    <![CDATA[]]>
  </system-err>
</testcase>
</testsuite>

```

Reply:
Thus, I believe that durations for cucumber hooks already included.

Reply:
OK, thanks